app/components/works/dates_component.html.erb

```diff
@@ -6,117 +6,116 @@
                                                  min_year: min_year,
                                                  max_year: max_year) %>
 
-  <div class="mb-3 row">
-    <div class="col-sm-3 h6">
-      Creation date
-      <%= render PopoverComponent.new key: 'work.creation_date' %>
-    </div>
-  </div>
+<fieldset>
 
-  <div class="mb-3 row" data-complex-radio-target="selection">
-    <div class="col-sm-2">
-      <%= form.radio_button :created_type, 'single', class: 'form-check-input', data: { action: 'complex-radio#disableUnselectedInputs' } %>
-      <%= form.label :created_type_single, 'Single date' %>
-    </div>
-    <div data-controller="date-validation" class="col-sm-10">
-      <div class="year">
-        <label for="<%= prefix %>_created_year" class="visually-hidden">Created year</label>
-        <%= number_field_tag "#{prefix}[created(1i)]", created_year,
-            data: { date_validation_target: 'year', action: 'date-validation#change' },
-            id: "#{prefix}_created_year",
-            placeholder: 'year',
-            class: "form-control", min: min_year, max: max_year %>
-        <div class="invalid-feedback" data-date-validation-target="error"></div>
-      </div>
-      <div class="month">
-        <label for="<%= prefix %>_created_month" class="visually-hidden">Created month</label>
-        <%= select_month created_month, { prefix: prefix, field_name: 'created(2i)', prompt: 'month'},
-                         data: { date_validation_target: 'month', action: 'date-validation#change' },
-                         id: "#{prefix}_created_month", class: "form-control" %>
-      </div>
+    <legend class="h5">Creation date <%= render PopoverComponent.new key: 'work.creation_date' %></legend>
```

Comment:
Only "phrasing content" (https://html.spec.whatwg.org/multipage/dom.html#phrasing-content) is permitted within `<legend>`. I don't think the `PopoverComponent` is permitted.